### ISSUE TITLE
About setting different learning rate in different layers

### ISSUE THREAD

**Issue by SimpleXP** [TuSimple/simpledet#153]
Hi,

I would like to set different learning rate for different layers. For examples, I use the default learning rate settings during training on dataset 1, now I need to train on dataset 2 and I want to keep the first few layers fixed. How do I set the learning rate of some layers to 0?
Thanks in advance if anyone can provide me some answers or hints!

**Comment by RogerChern** [comment]
Hi @SimpleXP,
The `ModelParam > pretrain > fix_params` should fit your need. 
If the name of your parameter **partially** matches any strings in `fix_params`, the parameter will be frozen. 
In the example below, you can see that the first conv, the first stage and all BN gamma beta are fixed as the standard setting.

https://github.com/TuSimple/simpledet/blob/a95605aa053640f9565551b94111a2bdf61d45d2/config/faster_r50v1c4_c5_512roi_1x.py#L130-L133

**Comment by SimpleXP** [comment]
> Hi @SimpleXP,
> The `ModelParam > pretrain > fix_params` should fit your need.
> If the name of your parameter **partially** matches any strings in `fix_params`, the parameter will be frozen.
> In the example below, you can see that the first conv, the first stage and all BN gamma beta are fixed as the standard setting.
> 
> https://github.com/TuSimple/simpledet/blob/a95605aa053640f9565551b94111a2bdf61d45d2/config/faster_r50v1c4_c5_512roi_1x.py#L130-L133

Thanks, Roger! Problem solved!
